core/frontend/src/WebViewerApp.ts

```diff
@@ -31,50 +29,50 @@ export interface WebViewerAppOpts {
  * iModels readonly, and may not use Ipc.
  * @beta
  */
-export class WebViewerApp {
+// export class WebViewerApp {

```

Comment:
@calebmshafer i think the main difference in the Web and Desktop Viewe pkgs was how e called IModelApp.Startup, web calling WevViewerApp.startup and Desktop same too. With this change are we goign back to consolidating so all apps needs to call IModelApp.startup?

Reply:
@pbell97 lets completely remove this.

Reply:
Can you update all docs that reference WebViewerApp as well?